hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/Gauges.java

```diff
@@ -18,5 +18,6 @@
 
             THREADS = "threads",
             INFLIGHT_REQUESTS = "inflight-requests",
-            OUTPUT_RATE = "output-rate." + GROUP + "." + TOPIC + "." + SUBSCRIPTION;
+            OUTPUT_RATE = "output-rate." + GROUP + "." + TOPIC + "." + SUBSCRIPTION,
+            BACKUP_STORAGE_MESSAGES = "backup-storage-messages-number";
```

Comment:
`backup-storage.size`? this way we can put some more metrics in `backup-storage` if we need in future